DOL.WHD.Section14c.Api/Controllers/SupportedFileTypesController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using DOL.WHD.Section14c.Business;
+using DOL.WHD.Section14c.Domain.Models.Submission;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+
+namespace DOL.WHD.Section14c.Api.Controllers
+{
+    /// <summary>
+    /// Controller for managing application attachment supported file types
+    /// </summary>
+    [RoutePrefix("/api/SupportedFileTypes")]
+    public class SupportedFileTypesController : ApiController
+    {
+        private readonly IAttachmentSupportedFileTypesService _supportedFileTypesService;
+
+        /// <summary>
+        /// Constructor to handle passed supported file types service
+        /// </summary>
+        /// <param name="supportedFileTypesService"></param>
+        public SupportedFileTypesController(IAttachmentSupportedFileTypesService supportedFileTypesService)
```

Comment:
I'm curious why the supported file types are stored in the database AND the app config.  It seems like they should only be in one place.  Why can't this controller (or the `AttachmentSupportedFileTypesService`, either one) just parse the string from `AppSettings`?

E.g., something like this might work:
```c#
Regex typesRegex = new Regex("^" + Regex.Escape("^.*\.(") + "([^)]+)" + Regex.Escape(")$"))
String[] types = typesRegex.match(AppSettings.Get<string>("AllowedFileNamesRegex")).Value.Split("|")
```

Not trying to prescribe a solution, just curious why we'd want to have this configuration information in two places.

Reply:
@dciceman where did we land with this? it was going to be one or the other.

Reply:
I will make the update to parse string from AppSettings.